.golangci.yml

```
@@ -0,0 +1,95 @@
+      linters:
+        - funlen
+        - cyclop
+        - gocognit
+        - goerr113
+        - gochecknoinits
+        - gochecknoglobals
+
+run:
+  modules-download-mode: readonly
+  timeout: 5m
+  allow-parallel-runners: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)